leakcanary-android/src/main/java/com/squareup/leakcanary/AndroidExcludedRefs.java

```diff
@@ -382,6 +382,18 @@
       excluded.instanceField("android.view.Choreographer$FrameDisplayEventReceiver",
           "mMessageQueue", true);
     }
+  },
+
+  EDITTEXT_BLINK_MESSAGEQUEUE {
+    @Override void add(ExcludedRefs.Builder excluded) {
+      // The EditText Blink of the Cursor is implemented using a callback and Messages, which
```

Comment:
Add a link to the AOSP bug (https://code.google.com/p/android/issues/detail?id=188551) and fix (https://android.googlesource.com/platform/frameworks/base/+/5b734f2430e9f26c769d6af8ea5645e390fcf5af%5E%21/)